packages/fuel-indexer-types/src/scalar.rs

```diff
@@ -0,0 +1,124 @@
+use bytes::Bytes;
+pub use fuel_types::{
+    Address, AssetId, Bytes32, Bytes4, Bytes64, Bytes8, ContractId, MessageId, Salt, Word,
+};
+pub use fuels::{
+    core::try_from_bytes,
+    types::{
+        bech32::{Bech32Address, Bech32ContractId},
+        Bits256, Identity, SizedAsciiString,
+    },
+};
```

Comment:
I'm not sure if these should be re-exported under `fuel-indexer-types::scalar` as I'd imagine that the scalar types would be used for GraphQL-specific things. Perhaps we could re-export the ones that don't have a match in `fuel-indexer-schema/src/base.graphql` under a different name?

In any case, I don't feel strongly about this; just a suggestion.

Reply:
Yeah, the `fuels` types were what prompted the suggestion. It can wait for a future PR.